### PR TITLE
Added support for tracking headers

### PR DESCRIPTION
**Pull request by ewoutp**
This PR adds support for 2 additional request headers that can be used to track CLI activity.

### CODE REVIEW

**Comment by marians** [comment]
Looking good to me. Just saying, since it's in "review" state. :)

File: status.go [hunk 1]
```diff
@@ -4,8 +4,8 @@ import (
 	"time"
 
 	"github.com/giantswarm/api-schema"
-	appdeftransf "github.com/giantswarm/app-def-transf-go" // prefix for swagger, nothing left to say
```
Comment (ewoutp) [inline status.go]: These changes are needed to make swapper (api) happy.

**Comment by ewoutp** [comment]
@zeisss RFR

**Comment by zeisss** [comment]
LGTM

**Pull request merged**